backends/gstreamer/webrtc.rs

```diff
@@ -571,23 +593,33 @@ impl GStreamerWebRtcController {
                 let thread_ = thread.lock().unwrap().clone();
                 match GStreamerWebRtcDataChannel::from(&id, channel, &thread_) {
                     Ok(channel) => {
+                        {
+                            let mut data_channels = data_channels.lock().unwrap();
+                            if let Some(ref mut channel) = data_channels.get_mut(&id) {
+                                match channel {
+                                    DataChannelEventTarget::Buffered(ref mut events) => {
+                                        for event in events.drain(0..) {
+                                            thread_.internal_event(
```

Comment:
Is there any chance that we could have a buffered close event? If so, we will end up registering a channel after the close.